association_backend/association/api/types.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Optional
+
+import strawberry
+from association.domain import entities
+
+
+@strawberry.type
+class Subscription:
+    user_id: str
+    creation_date: datetime
+    state: str
+    stripe_session_id: str
+    payment_date: Optional[datetime]
```

Comment:
I don't think payment_date should live here, Subscription should have a list of payments

Reply:
i agree with you, this will be updated soon